src/components/AVPlayer/AVPlayer.js

```diff
@@ -225,7 +225,7 @@ class AVPlayer extends Component {
     }
     window.removeEventListener('message', this.receiveMessageFunc, false);
     if (this.isUnitExistAndPlaying()) {
-      this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props.media));
+      this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props));
```

Comment:
Same questions as above.

Reply:
Good point, taking all params now from this.